web/webpack.config.js

```diff
@@ -11,12 +11,13 @@ const Define = webpack.DefinePlugin;
 const BundleAnalyze = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 
 const babelOptions = require('./babelOptions');
-const { NODE_ID } = require('./app/common/constants');
 const publicFolder = path.resolve(__dirname, 'public');
-const env = process.env.NODE_ENV || 'dev';
-
+const env = process.env.NODE_ENV || 'development';
+const remarkUrl = process.env.REMARK_URL || 'https://demo.remark42.com';
+const NODE_ID = 'remark42';
```

Comment:
process.env.REMARK_NODE?

Reply:
Ah, I see, sorry.

Reply:
Yeah, I had some troubles because of this unnecessary import from constants.js.
Now it is defined here and passed into DefinePlugin.